dev/VSIX/Extension/Original/WindowsAppSDK.Extension.csproj

```diff
@@ -54,11 +54,11 @@
     <Reference Include="System" />
   </ItemGroup>
   <ItemGroup>
```

Comment:
Just going off of names...:

We create these VSIXs:
WindowsAppSDK.Cpp.Extension.Dev16.Experimental.vsix
WindowsAppSDK.Cpp.Extension.Dev17.Experimental.Component.vsix
WindowsAppSDK.Cpp.Extension.Dev17.Experimental.Standalone.vsix
WindowsAppSDK.Cs.Extension.Dev16.Experimental.vsix
WindowsAppSDK.Cs.Extension.Dev17.Experimental.Component.vsix
WindowsAppSDK.Cs.Extension.Dev17.Experimental.Standalone.vsix
WindowsAppSDK.Extension.Experimental.vsix

Based on those names and the names of the 5 CSPROJ files  under dev\vsix\Extension, I suspect this file corresponds to WindowsAppSDK.Extension.Experimental.vsix. Assuming I'm right, given that WindowsAppSDK.Extension.Experimental.vsix targets both Dev16 and 17, should the version updates in this file be done? The version updates are only wanted for Dev16, right?

Reply:
Technically, the WindowsAppSDK.Extension.Experimental is about to be removed. It's only there as a placeholder until we release 1.0 to the marketplace and replace it with C++ and C# specific flavors.

Reply:
Got it. Still, better to leave it as-is so we don't break the VS 2022 scenario, no? I say that because devs like me may use the WindowsAppSDK.Extension.Experimental.vsix on VS2022 without knowing.

Reply:
Also, the version-less extension targets both Dev16 and Dev17. And in order to target Dev17 at all, it needs to use the Dev17 version of the VS SDK.

Reply:
I made another mistake... My comment above should have read:
"Got it. Still, better to leave it as-is so we don't break the VS 2019 scenario, no? I say that because devs like me may use the WindowsAppSDK.Extension.Experimental.vsix on VS2019 without knowing."

> in order to target Dev17 at all, it needs to use the Dev17 version of the VS SDK.

Got it. What will happen in the Dev16 case?

Reply:
It should continue to work, as-is, because the versionless extension doesn't have any localization or code in it.

I'd recommend not using it though, because it's not an accurate representation of what our customers will be using. Again, the *only* reason I haven't yet removed this project, is because we'll probably want to publish one last update to "deprecate" it and point our customers towards the new replacement extensions. As soon as we do that, I'm deleting it.

Reply:
OK, thanks for explaining.